docker/app-api/Dockerfile

```
@@ -56,4 +56,5 @@ EXPOSE 8080
 
 USER 11020
 
-CMD ["java","-Djava.security.egd=file:/dev/./urandom","-jar","/opt/app/app.jar"]
+# Netty connection parameters come from `System.getProperty(...)`(resolution: ms).
+CMD ["java","-Djava.security.egd=file:/dev/./urandom","-Dreactor.netty.pool.maxIdleTime=20000","-Dreactor.netty.pool.maxLifeTime=60000","-jar","/opt/app/app.jar"]
```

```suggestion
CMD ["java","-Djava.security.egd=file:/dev/./urandom","-Dreactor.netty.pool.maxIdleTime=$MAX_IDLE_TIME","-Dreactor.netty.pool.maxLifeTime=$MAX_LIFE_TIME","-jar","/opt/app/app.jar"]
```

I reckon Dave meant linux environment variables rather than being done through `application.yml`.

---

Thanks @pori ,  You are correct, I found the configuraiton is a system configuration 
```
[Add System properties for connection pool - max idle time and leasing…](https://github.com/reactor/reactor-netty/commit/0c38ed4dcd5e8b83fc749620ba23a9cf75074368) …
```
so it can't be done via `application.yaml`

And we need to make the time can be configured outside the docker via the linux env variables.

cc @jackworkgit

---

I found a new method to change these parameters in `application.yml`. I will try first and then submit it later.